### PR TITLE
Add testing GitHub action

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: "Testing"
```
Comment (MartinHjelmare) [inline .github/workflows/testing.yml]: I'd call it `"Test"` in imperative. Same for the workflow file.

File: test/plugins/LabelBot/label_bot.spec.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ describe("LabelBotPlugin", () => {
       ],
     });
     assert.deepEqual(setLabels, {
-      labels: ["integration: mqtt", "merging-to-master", "core"],
+      labels: ["integration: mqtdt", "merging-to-master", "core"],
```
Comment (MartinHjelmare) [inline test/plugins/LabelBot/label_bot.spec.ts]: What are we testing? Should it be a missing integration?

Comment (ludeeus) [reply test/plugins/LabelBot/label_bot.spec.ts]: uppppps, that should not be there.

File: .github/workflows/testing.yml (hunk 1 above)
Comment (ludeeus) [reply .github/workflows/testing.yml]: Changed 👍

**Review by MartinHjelmare — APPROVED** [review]
Nice!

**Pull request merged**